web/themes/custom/cloudy/scss/_bootstrap-overrides.scss

```diff
@@ -59,3 +60,11 @@ $nav-pills-link-active-bg: #000;
 // chosen
 $chosen-sprite-path: '/themes/custom/cloudy/images/chosen-sprite.png';
 $chosen-sprite-retina-path: '/themes/custom/cloudy/images/chosen-sprite@2x.png';
+
+// Horizontal rule
+
+$hr-border-color: $light-gray;
+$hr-margin-y: 1rem;
+$hr-margin-y: 1rem;
+$hr-border-width: 3px;
+$hr-border-color: $gray;
```

Comment:
@julegan, I see a small issue/quick fix. $hr-border-color is declared twice.

Reply:
Fixed. It is rebuilding without the duplicate border color and margin.

Reply:
Thanks for catching that. One small thing, the color choice was meant to be `$hr-border-color: $gray;`.